logging/telemetry_test.go

```diff
@@ -18,6 +18,8 @@ package logging
 
 import (
 	"fmt"
+	"github.com/algorand/go-algorand/config"
```

Comment:
nit: coding style - the import list should be grouped as follows:
```
import (
 	"fmt"
 	"os"
 	"testing"
 	"time"

	"github.com/sirupsen/logrus"
	"github.com/stretchr/testify/require"

	"github.com/algorand/go-deadlock"

 	"github.com/algorand/go-algorand/config"
	"github.com/algorand/go-algorand/logging/telemetryspec"
)
```

Reply:
Thanks! I just fixed that